cmd/zdb/zdb.c

```diff
@@ -4481,8 +4495,10 @@ main(int argc, char **argv)
 		args.path = searchdirs;
 		args.can_be_active = B_TRUE;
 
-		error = zpool_tryimport(g_zfs, target, &cfg, &args);
+		error = zpool_tryimport(g_zfs, target_pool, &cfg, &args);
```

Comment:
It's a little concerning we didn't have any test coverage for this.  What do you think about adding a very basic testcase to `tests/zfs-tests/tests/functional/cli_root/zdb/` which covers passing the pool/dataset for imported/exported pools.

Reply:
Will do.